demos/meson.build

```diff
@@ -22,10 +22,16 @@ executable('gears', 'gears.c', vkd3d_headers,
   dependencies        : demo_vkd3d_deps,
   install             : true,
   gui_app             : true,
-  override_options    : [ 'c_std='+vkd3d_c_std ])
+  override_options    : [
+    'c_std='         + vkd3d_c_std,
+    'warning_level=' + vkd3d_warning_level
+  ])
```

Comment:
is there no way to set these globally? at least c_std should be a project option.

Reply:
Ah hah, we can!

Reply:
Done.